### PR TITLE
fix: autoprovision with group timestamp false

### PR DESCRIPTION
**Pull request by AlvaroVega**
typeInformation should contain that timestamp=false in an autoprovision of device with a group with timestamp=false

Related with issue https://github.com/telefonicaid/iotagent-node-lib/issues/1504 and the expected new tests about it

### CODE REVIEW

File: lib/services/devices/deviceService.js [hunk 1]
```diff
@@ -173,6 +173,9 @@ function mergeDeviceWithConfiguration(fields, defaults, deviceData, configuratio
     if (configuration && configuration.entityNameExp !== undefined) {
```
Comment (fgalan) [inline lib/services/devices/deviceService.js:173]: Changes already included in existing CNR entry?

Comment (AlvaroVega) [reply lib/services/devices/deviceService.js:173]: fixed in https://github.com/telefonicaid/iotagent-node-lib/pull/1508/commits/93c6952ed7e740ec4bcbf544e05de4c018342a5b

**Comment by AlvaroVega** [comment]
Do not merge

**Comment by AlvaroVega** [comment]
As is this PR makes all autoprovisioned devices would be created with timestmap configuration from group. Not sure if we want it.

**Comment by AlvaroVega** [comment]
Currently for explicitAttrts and autorpovision device inherits value for explicitAttrs

**Comment by AlvaroVega** [comment]
related issue https://github.com/telefonicaid/iotagent-node-lib/issues/1377

File: CHANGES_NEXT_RELEASE [hunk 2]
```diff
@@ -1,3 +1,4 @@
+- Fix include timestamp from group in device autoprovisioned
```
Comment (fgalan) [inline CHANGES_NEXT_RELEASE:1]:
```suggestion
- Fix: include timestamp from group in device autoprovisioned
```

No associated issue?

**Review by mapedraza — APPROVED** [review]
LGTM

File: CHANGES_NEXT_RELEASE [hunk 3]
```diff
@@ -1,3 +1,4 @@
+- Fix: use but no store timestamp and explicitAttrs from group with autoprovisioned devices
```
Comment (fgalan) [inline CHANGES_NEXT_RELEASE:1]:
```suggestion
- Fix: use but not store timestamp and explicitAttrs from group with autoprovisioned devices (#1504, partially)
```

**Review by fgalan — APPROVED** [review]
LGTM

**Pull request merged**